src/scripts/constants.ts

```diff
@@ -339,10 +352,14 @@ export module Constants {
 	}
 
 	export module Settings {
-		export var timeBetweenTooltips = 1000 * 60 * 60 * 24 * 7 * 3; // 21 days
-		export var noOpTrackerTimeoutDuration = 20 * 1000; // 20 seconds
+		export var fontSizeStep = 2;
+		export var maxClipSuccessForRatingsPrompt = 12; // TODO waiting for consensus on this value
+		export var maximumJSTimeValue = 8640000000000000; // http://ecma-international.org/ecma-262/5.1/#sec-15.9.1.1
```

Comment:
sorry, what would the multiplication you're suggesting here be? do you find 100,000,000 (days) * 86,400,000 (ms) more readable (as described by the spec linked to)?